### PR TITLE
feature/drf-3.9-support

### PR DESCRIPTION
**Pull request by manish-lamichhane**
- Drops obsolete @detail_route decorator
- Adds @action decorator

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -52,3 +52,6 @@ docs/_build/
 
 # PyBuilder
 target/
+
+# IDE
+.idea/
```
Comment (cwuebbels) [inline .gitignore]: Personal staff should not part of this project

File: drf_fsm_transitions/viewset_mixins.py [hunk 2]
```diff
@@ -22,7 +22,7 @@ def inner_func(self, request, pk=None, **kwargs):
     return inner_func
 
 
-def get_viewset_transition_action_mixin(model, **kwargs):
+def get_viewset_transition_action_mixin(model, field='status', **kwargs):
```
Comment (cwuebbels) [inline drf_fsm_transitions/viewset_mixins.py]: field='status' should be field='state' (default value is state)

Comment (manish-lamichhane) [reply drf_fsm_transitions/viewset_mixins.py]: updated

File: .gitignore (hunk 1 above)
Comment (manish-lamichhane) [reply .gitignore]: Done

**Review by cwuebbels — APPROVED** [review]

**Pull request merged**